mantidproject/mantid
# Paraview Purge as part of Qt4 Removal

**Description of work.**
I am removing Paraview, VSI and Surrounding supported code. This includes DynamicPDF Interface. This is a part of the removal of Qt4


**To test:**
Ensure Qt help docs open on MacOS and Windows
Ensure Packaging works and installs properly of MacOS and Windows

Ubuntu and RHEL7/CentOS are tested by Builds and myself.


Fixes #30500 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
